apps/examples/bluetooth/Makefile

```diff
@@ -20,6 +20,8 @@
 -include $(TOPDIR)/Make.defs
 include $(APPDIR)/Make.defs
 
+CFLAGS += -I$(TOPDIR)/../framework/include/bluetooth
```

Comment:
it's not recommended to add include path like this way.
you can refer include path by modifying source files
System contained framework/include path

Reply:
Done.